packages/fhir-care-team/src/components/ViewDetails/tests/index.test.tsx

```diff
@@ -86,6 +87,9 @@ test('works correctly', async () => {
 });
 
 test('Closes on clicking cancel (X) ', async () => {
+  const history = createMemoryHistory();
+  history.push(URL_CARE_TEAM);
+
```

Comment:
if this was intentional, what does it add to the test?

Reply:
Previously, on clicking the cancel button of the view-details, the onclick just triggered a history.push to this url. Changing to useParamsHook this changes to where we use the removeParam function which just removes the querystring from the url hence the need to simulate the history behaviour in our test to get the same thing.